gym/lib/gym/error_handler.rb

```diff
@@ -104,6 +104,10 @@ def handle_empty_archive
         print "Also, make sure to have a valid code signing identity and provisioning profile installed"
         print "Follow this guide to setup code signing https://docs.fastlane.tools/codesigning/GettingStarted/"
         print "If your intention was only to export an ipa be sure to provide a valid archive at the archive path."
+        print "This error might also happen if your workspace/project file is not in the root directory of your project"
+        print "To workaround that issue, you can add `Dir.chdir('../path/to/dir/containing/proj') do`"
```

Comment:
@KrauseFx  I was wondering where you would add `Dir.chdir` because it's not really clear here. I assume it's in the Fastfile.

@orta yeah, I think that's a really great idea too. I'd also leave in the specific commit that was mentioned here about checking out your example.